vmdk_plugin/utils/test_util/VolumeCreationTestUtil.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This util is going to hold various helper methods to be consumed by testcase.
+// Volume creation, deletion is supported as of now.
+ 
+package e2e
+
+import (
+	"os/exec"
+	"os"
+	"fmt"
+	"strings"
+)
+
+// This util method is going to create vsphere docker volume with 
+// defaults.
+func CreateDefaultVolume(ip string, name string) ([]byte, error) {
+        
+	fmt.Printf("\ncreating volume [%s] on VM[%s]", name, ip)
+
+        return exec.Command("/usr/bin/ssh", strings.Split(os.Getenv("SSH_KEY_OPT"), " ")[0], strings.Split(os.Getenv("SSH_KEY_OPT"), " ")[1], "-q", "-kTax", "-o StrictHostKeyChecking=no", "root@"+ip, "docker volume create --driver=vmdk --name="+name).CombinedOutput()
+
+}
+
+// This helper method is going to create vsphere docker volume with 
+// defaults and verifies as well
+func CreateDefaultVolumeAndVerify (ip string, name string) ([]byte, error, bool) {
```

Comment:
one of the method simply creates volume and returns control to testcase to process error whether fails the testcase or ignores the error occurred while executing request. 

```CreateDefaultVolumeAndVerify``` processed the err field and on that basis it returns bool for testcase to proceed further.